gm4_zauber_liquids/data/gm4_zauber_liquids/functions/liquid_init/harming.mcfunction

```diff
@@ -1,4 +1,4 @@
-data merge block ~ ~ ~ {CustomName:'"Zauber Harming Potion Tank"'}
+data merge block ~ ~ ~ {CustomName:'{"translate":"%1$s%3427655$s","with":["Zauber Harming Potion Tank", {"translate":"block.gm4.zauber_liquids.harming"}]}'}
```

Comment:
To be consistent with potion tanks, the translation key should be `block.gm4.zauber_tank.harming`. See https://github.com/Gamemode4Dev/GM4_Resources/blob/master/gm4_translations/assets/gm4/lang/en_us.json#L111